roles/aodh/templates/etc/aodh/aodh.conf

```diff
@@ -5,6 +5,8 @@ rpc_backend = rabbit
 log_dir = /var/log/aodh
 debug = {{ aodh.debug }}
 
+rest_notifier_ssl_verify = False
```

Comment:
without this, seems aodh-notifier (old as ceilometer-alarm-notifer) have error log about SSL when try to http callback to heat, and this config was used in old ceilometer.conf

[Fixed Heat Autoscaling By Making Ceilometer Alarms Not Verify SSL](https://github.com/blueboxgroup/ursula/commit/6e75494d6458ed2229fe4a6e3b2e7c9e6a3c09bc)